### ISSUE TITLE
F4SE + ENB

### ISSUE THREAD

**Issue by RestedWicked** [TanninOne/modorganizer#808]
anyone managed to get f4se+enb/reshade working?
I can run one or the other. trying to run both the game doesnt make it to the menu

**Comment by TanninOne** [comment]
F4se+enb works fine for me. Please provide details: OS, a log file, dmp if any, error message if you get one,...

**Comment by TanninOne** [comment]
I meant the usvfs log

**Comment by RestedWicked** [comment]
As soon as I get home from work I'll try and send a log.
Using Windows10. And actually it's  f4se+enb+reshade+shadowboost
I don't get any error msg. The game starts and CTDs when about to load the main menu.
I can load the game with shadowboost+enb/reshade. I can also load the game with f4se+shadowboost. TBH until this moment I forgot I was running shadowboost and didn't try only f4se+enb. Gonna try it tonight.
I remember back in alpha2 there was problems with shadowboost+enb but I couldn't find the thread.

**Comment by RestedWicked** [comment]
update: the problem is trying to use f4se+enb+shadowboost. I can run any 2 of them together but can't run all 3

**Comment by Reversao** [comment]
I'm having a similar issue using reshade+F4SE causes fallout 4 to crash when launched from MO2 2.0.6. I turned off all my mods and still will crashed. If launched from Nexus fallout 4 will load just fine. 

I'm having a similar issue using reshade+F4SE causes fallout 4 to crash when launched from MO2 2.0.6. I turned off all my mods and still will crashed. If launched from Nexus fallout 4 will load just fine. 

Windows 7 64 Bit

 Problem Event Name:	BEX64
  Application Name:	Fallout4.exe
  Application Version:	1.5.307.0
  Application Timestamp:	5751d3cb
  Fault Module Name:	StackHash_578f
  Fault Module Version:	0.0.0.0
  Fault Module Timestamp:	00000000
  Exception Offset:	0000000000000000
  Exception Code:	c0000005
  Exception Data:	0000000000000008
  OS Version:	6.1.7601.2.1.0.256.48
  Locale ID:	1033
  Additional Information 1:	578f
  Additional Information 2:	578fe98623d4ddb6b61de76532f618ea
  Additional Information 3:	cf62
  Additional Information 4:	cf624389e011891810c289ed07500a62

usvfs log:
http://pastebin.com/EFPktdN7
